spring-boot-project/spring-boot-docs/src/main/kotlin/org/springframework/boot/docs/actuator/metrics/supported/mongodb/command/MyCommandTagsProviderConfiguration.kt

```diff
@@ -24,8 +24,6 @@ import org.springframework.context.annotation.Configuration
 class MyCommandTagsProviderConfiguration {
 
 	@Bean
-	fun customCommandTagsProvider(): MongoCommandTagsProvider? {
-		return CustomCommandTagsProvider()
-	}
+	fun customCommandTagsProvider() = CustomCommandTagsProvider()
```

Comment:
I'm not a fan of this change, as this removes the explicit `MongoCommandTagsProvider` return type.

Reply:
done.